app/views/galleries/show.rb

```diff
@@ -15,6 +15,20 @@ def js_vars
       [{ name: 'pageName', value: 'collections/galleries' }]
     end
 
+    def head_meta
+      mustache[:head_meta] ||= begin
+        description = @gallery.description.strip
+        head_meta = gallery_head_meta + [
+          { meta_name: 'description', content: description },
+          { meta_property: 'og:description', content: description },
+          { meta_property: 'og:image', content: gallery_items_content.first[:full_url] }
+        ]
+        head_meta << { meta_property: 'og:title', content: @gallery.title } unless @gallery.title.nil?
+        head_meta << { meta_property: 'og:sitename', content: @gallery.title } unless @gallery.title.nil?
```

Comment:
```
{ meta_property: 'og:title', content: @gallery.title },
{ meta_property: 'og:sitename', content: @gallery.title } 

```
can go up into the array, and then we can get rid of the head_meta variable altogether.
no need for the nil? checks. so super gets added to then end of the array like:
```
gallery_head_meta + [
 ...
] + super
```